src/match.js

```diff
@@ -153,26 +153,29 @@ function findAttributesPattern (priority, element, ignore, exclude) {
 
     const currentIgnore = ignore[attributeName] || ignore.attribute
     const currentDefaultIgnore = defaultIgnore[attributeName] || defaultIgnore.attribute
+
     if (checkIgnore(currentIgnore, attributeName, attributeValue, currentDefaultIgnore)) {
       continue
     }
 
     var pattern = `[${attributeName}="${attributeValue}"]`
 
-    if ((/\b\d/).test(attributeValue) === false) {
-      if (attributeName === 'id') {
-        pattern = `#${attributeValue}`
-      }
+    // this if is here commented because we have a better validation for this cases.
+    // if ((/\b\d/).test(attributeValue) === false) {
```

Comment:
This should solve the optimisation/ignore issues for class names that were left as `[class="all classes here"]` 👍